### PR TITLE
zh-cn: update the translation of WebAssembly landing page

### PR DESCRIPTION
**Pull request by yin1999**
### Description

update the translation of WebAssembly landing page

### CODE REVIEW

File: files/zh-cn/webassembly/index.md [hunk 1]
```diff
@@ -1,77 +1,93 @@
...
 ## 简而言之
 
-对于网络平台而言，WebAssembly 具有巨大的意义——它提供了一条途径，以使得以各种语言编写的代码都可以以接近原生的速度在 Web 中运行。在这种情况下，以前无法以此方式运行的客户端软件都将可以运行在 Web 中。
+对于 Web 平台而言，WebAssembly 具有巨大的意义——它提供了一条使得以各种语言编写的代码都可以接近原生的速度在 Web 中运行的途径，使得以前无法在 Web 上运行的客户端应用程序得以在 Web 上运行。
 
-WebAssembly 被设计为可以和 JavaScript 一起协同工作——通过使用 WebAssembly 的 JavaScript API，你可以把 WebAssembly 模块加载到一个 JavaScript 应用中并且在两者之间共享功能。这允许你在同一个应用中利用 WebAssembly 的性能和威力以及 JavaScript 的表达力和灵活性，即使你可能并不知道如何编写 WebAssembly 代码。
+WebAssembly 被设计为可以和 JavaScript 一起协同工作——通过使用 WebAssembly 的 JavaScript API，你可以把 WebAssembly 模块加载到 JavaScript 应用中并且在两者之间共享功能。这允许你在同一个应用中利用 WebAssembly 的性能和能力以及 JavaScript 的表达力和灵活性，即使你可能并不知道如何编写 WebAssembly 代码。
 
-而且，更棒的是，这是通过 [W3C WebAssembly Community Group](https://www.w3.org/community/webassembly/) 开发的一项网络标准，并得到了来自各大主要浏览器厂商的积极参与。
+而且，更棒的是，这是由 [W3C WebAssembly 社区组](https://www.w3.org/community/webassembly/)开发的 Web 标准，并得到了来自各大主要浏览器厂商的积极参与。
```
Comment (jasonren0403) [inline files/zh-cn/webassembly/index.md:18]: 少了个 W3C WebAssembly Working Group

**Review by jasonren0403 — COMMENTED** [review]
some suggestions

File: files/zh-cn/webassembly/index.md (hunk 1 above)
Comment (yin1999) [reply files/zh-cn/webassembly/index.md:18]:
```suggestion
而且，更棒的是，这是由 [W3C WebAssembly 工作组](https://www.w3.org/wasm/)和[社区组](https://www.w3.org/community/webassembly/)开发的 Web 标准，并得到了来自各大主要浏览器厂商的积极参与。
```

**Review by jasonren0403 — APPROVED** [review]
LGTM 👍

**Pull request merged**